pack/td.json

```diff
@@ -472,4 +472,38 @@
         "type_code": "asset",
         "uniqueness": false
     }
+    {
+        "code": "13056",
+        "cost": 3,
+        "deck_limit": 3,
+        "faction_code": "neutral-corp",
+        "faction_cost": 0,
+        "illustrator": "Shawn Ye Zhongyi",
+        "keywords": "Code Gate",
+        "pack_code": "td",
+        "position": 56,
+        "quantity": 3,
+        "side_code": "corp",
+        "strength": 3,    
+        "text": "[subroutine] The Runner loses [click], if able\n[subroutine] The Runner trashes 1 card from his or her grip.",
+        "title": "Weir",
+        "type_code": "ice",
+        "uniqueness": false
+}{
```

Comment:
missing "," separator
new object should start on a new line, and be indented correctly